regression-test/suites/partition_p0/multi_partition/test_multi_partition.groovy

```diff
@@ -321,4 +321,37 @@ suite("test_multi_partition") {
     assertEquals(result2.size(), 16)
     sql "drop table multi_par8"
 
+
+    sql "drop table if exists multi_par9"
+    sql """
+        CREATE TABLE IF NOT EXISTS multi_par9 (
+            k1 tinyint NOT NULL,
+            k2 smallint NOT NULL,
+            k3 int NOT NULL,
+            k4 bigint NOT NULL,
+            k5 decimal(9, 3) NOT NULL,
+            k6 char(5) NOT NULL,
+            k10 date NOT NULL,
+            k11 datetime NOT NULL,
+            k12 datev2 NOT NULL,
+            k13 datetimev2 NOT NULL,
+            k14 datetimev2(3) NOT NULL,
+            k15 datetimev2(6) NOT NULL,
+            k7 varchar(20) NOT NULL,
+            k8 double max NOT NULL,
+            k9 float sum NOT NULL )
+        AGGREGATE KEY(k1,k2,k3,k4,k5,k6,k10,k11,k12,k13,k14,k15,k7)
+        PARTITION BY RANGE(k3) (
+            FROM (1000) TO (10000) INTERVAL 200
```

Comment:
What if `FROM (1000) TO (1001) INTERVAL 200`?

Reply:
build one partition : [1000,1001)。
```
            PartitionValue lowerPartitionValue = new PartitionValue(beginNum);
            beginNum += interval;
            Long currentEnd = Math.min(beginNum, endNum);
            PartitionValue upperPartitionValue = new PartitionValue(currentEnd);
```